src/main/osd/osd_elements.c

```diff
@@ -1082,12 +1084,13 @@ static void osdElementEfficiency(osdElementParms_t *element)
 {
     int efficiency = 0;
     if (sensors(SENSOR_GPS) && ARMING_FLAG(ARMED) && STATE(GPS_FIX) && gpsSol.groundSpeed >= EFFICIENCY_MINIMUM_SPEED_CM_S) {
-        const int speedX100 = osdGetSpeedToSelectedUnit(gpsSol.groundSpeed * 100); // speed * 100 for improved resolution at slow speeds
+        const int speedX1 = osdGetSpeedToSelectedUnit(gpsSol.groundSpeed); 
         
-        if (speedX100 > 0) {
+        if (speedX1 > 0) {
             const int mAmperage = getAmperage() * 10; // Current in mA
-            efficiency = mAmperage * 100 / speedX100; // mAmperage * 100 to cancel out speed * 100 from above
+            efficiencyF += EFFFILTERTC * (mAmperage / speedX1 - efficiencyF); 
         }
+        efficiency = (int)round(efficiencyF);
```

Comment:
Please use `lrintf` here. `round` uses doubles and we have to avoid it.

Reply:
Fixed!